paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: 'bbousuite: A set of R packages to facilitate analysis of Boreal Caribou survival and recruitment data'
+authors:
+- name: Seb Dalgarno
+  orcid: 0000-0002-3658-4517
+  affiliation: 1
+- name: John Boulanger
+  orcid: 0000-0001-8222-1445
+  affiliation: 2
+- name: Ayla Pearson
+  orcid: 0000-0001-7388-1222
+  corresponding: true
+  affiliation: 1
+- name: Joe Thorley
+  orcid: 0000-0002-7683-4592
+  affiliation: 1
+- name: Troy Hegel  
+  affiliation: 3
+- name: Barry Nobert
+  affiliation: 3
+- name: Dave Hervieux
+  affiliation: 3
+affiliations:
+- index: 1
+  name: Poisson Consulting, Canada
+- index: 2
+  name: Integrated Ecological Research, Canada
+- index: 3
+  name: Province of Alberta, Canada
+date: 6 May 2024
+bibliography: paper.bib
+output: 
+  rticles::joss_article: default
+csl: apa.csl
+journal: JOSS
+tags:
+  - boreal caribou
+  - Bayesian
+  - maximum likelihood
+  - survival
+  - recruitment
+  - population growth
+---
+
+# Summary
+
+Although many animal populations are managed based on their abundance [@williams_analysis_2002], reliable abundance estimates can be impractical to obtain for mobile herding populations with large ranges such as boreal caribou (*Rangifer tarandus caribou*).
+Consequently, such populations are commonly managed based on their population growth rate ($\lambda$), which can be estimated from the population recruitment and adult female survival rates [@hatter_moose_1991]. 
+
+The `bbou` suite is a set of R packages [@r_core_team_2023] to facilitate simulation and analysis of boreal caribou survival and recruitment data to estimate population growth.  
```

Comment:
We need to update to R 2024 for the citation.

Reply:
done